apps/src/blockly/addons/functionEditor.js

```diff
@@ -168,14 +168,14 @@ export default class FunctionEditor {
           NAME: procedure.getName(),
         },
         deletable: false,
-        movable: false,
```

Comment:
I moved this down because it's one of the values I reset in `getCode`, and I wanted to put it next to the x/y position that are also reset/specific to the editor.